modules/cli/docs/generated/galasactl_runs_update.md

```diff
@@ -0,0 +1,35 @@
+## galasactl runs update
+
+Update tags for a named test run.
+
+### Synopsis
+
+Update tags for a named test run.
+
+```
+galasactl runs update [flags]
+```
+
+### Options
+
+```
+      --add-tags strings      comma-separated list of tags to add
```

Comment:
Can the --add-tags flag also be supplied like:
```
--add-tags tag1 --add-tags tag2
```

If so, it would be good to make it clear that you can either supply a comma-separated list or multiple `--add-tags` flags